lib/rules/require-aligned-object-values.js

```diff
@@ -12,10 +40,13 @@ module.exports.prototype = {
             'skipWithFunction': 'ignoreFunction',
             'skipWithLineBreak': 'ignoreLineBreak'
         };
+
```

Comment:
I'm checking what empty lines I've added with `git show`, `git show HEAD~2` or `git diff origin/master` (after rebasing of origin/master) and just removing random spaces I've accidentally added: to make commits focused on issues.